plugins/coverbrowser.koplugin/main.lua

```diff
@@ -148,6 +148,11 @@ function CoverBrowser:addToMainMenu(menu_items)
             checked_func = function() return G_reader_settings:readSetting("autoremove_deleted_items_from_history") end,
             callback = function() G_reader_settings:flipNilOrFalse("autoremove_deleted_items_from_history") end,
         },
+        {
+            text = _("Show filename in Open last menu item"),
```

Comment:
Note that there's presumably some duplication here, unless this is in the wrong spot altogether.

Reply:
Can you be clearer? (what's duplicated? the wrong spot for what? talking about the feature or my english ? :)

Reply:
Nothing's duplicated here, which is exactly the problem. Things are duplicated in CoverBrowser.

Reply:
huh? can you explain?

Reply:
I assumed all such things were duplicated, but apparently in this case it's not.

https://github.com/koreader/koreader/blob/9e4e71d63016d520eab1d59cc07e644958c63afe/plugins/coverbrowser.koplugin/main.lua#L85-L86

If that is an issue it's otherwise mostly unrelated to this PR, assuming this setting shouldn't be somewhere different.

Reply:
That's not an issue. When I wanted to add a few of the manual settings to the menus, they found their place in that submenu set up by coverbrowser. At the time when coverbrowser was not available on android, I added this in 2 part: the not full-featured part, with a 2 core settings that would work without coverbrowser (and it's with them that this PR new setting is added) - and the other coverbrowser settings for the full featured part.
So, they are not set up in another part of the code: only coverbrowser does it, no need to update some other place.

So yes, we could want to move these core settings outside coverbrowser (I don't, really), and when/if we do, we'll move 3 menu items instead of 2. Yet unrelated to this PR.

Reply:
Guys maybe new ticket for this?
So it won't be forgotten and this pr could be finished?